clients/src/test/java/org/apache/kafka/common/config/AbstractConfigTest.java

```diff
@@ -508,6 +509,55 @@ public void testDocumentationOfExpectNull() {
         assertNull(config.documentationOf("xyz"));
     }
 
+    @Test
+    public void testConcurrentUnusedUse() throws InterruptedException {
```

Comment:
I'm not sure this test adds much value from here but it picks up the issue and takes about 1.5s to run so I am happy to leave it in or remove it.

Reply:
I think I'd remove it or at least tag it as an integration test.

Reply:
I don't think it adds much value so I'd rather delete it.

Reply:
Done.